### ISSUE TITLE
Settings icons are missing

### ISSUE THREAD

**Issue by sugdyzhekov** [plesk/ext-route53#32]
Subj is present on Mass Update screen:
<img width="737" alt="screenshot 2018-10-14 at 19 16 30" src="https://user-images.githubusercontent.com/8955225/46920286-3e583600-cfec-11e8-889f-dde8d777bb67.png">

```
Product version: Plesk Onyx 17.9.6
     Build date: 2018/09/21 18:00
     OS version: CentOS 7.5.1804
       Revision: 485aa51d1d81a0c71683a67a3e259feb7ed61839
   Architecture: 64-bit
Wrapper version: 1.2
```
Russian locale, `sunny-day` skin

**Comment by ayamshanov** [comment]
Hi sugdyzhekov,

Thank you for the report. I have confirmed this bug. The bug id is EXTPLESK-681.

**Comment by ayamshanov** [comment]
Fixed since 2.7.2 (https://github.com/plesk/ext-route53/commit/d47d718bbc857bcbfa518d85748826069a748cb2)
